tests/test_errors.py

```diff
@@ -126,22 +129,12 @@ def test_repr(self):
     vf = ValidationFailed("vf_desc", ["vf"], [failure])
     assert repr(vf) == "ValidationFailed('vf_desc', ['vf'], [Failure('code', 'desc', ['a', 'b'])])"
 
-  def _assert_query_error(self, q, code, position=None):
+  def _assert_query_error(self, exception_class, q, code, position=None):
```

Comment:
I think `self, q, exception_class, code, position` flows better when calling this.